### ISSUE TITLE
Default to babel-preset-env current 

### ISSUE THREAD

**Issue by graingert** [danielmoore/ts-babel-node#11]

**Comment by orta** [comment]
Can you explain a little about what you mean by this? (sorry if it might seem obvious, but consider me a bit fresh to it all)

**Comment by graingert** [comment]
Currently you've got:

```js
var defaultBabelOpts = {
  presets: [ require('babel-preset-es2015') ],
};
```

Should be babel-preset-env

**Comment by alloy** [comment]
Fixed by #14
